ast_decompiler/decompiler.py

```diff
@@ -229,6 +238,11 @@ def generic_visit(self, node):
         raise NotImplementedError('missing visit method for %r' % node)
 
     def visit_Module(self, node):
+        # Check for docstring.
+        if isinstance(node.body[0], ast.Expr) and isinstance(node.body[0].value, ast.Str):
+            docstring = node.body.pop(0).value.s
```

Comment:
`decompile` shouldn't mutate the AST it is visiting; that can have unexpected effects.